Gem/Settings/ViewModels/ChainNodeViewModel.swift

```diff
@@ -2,23 +2,88 @@
 
 import Foundation
 import Primitives
+import BigInt
+import Components
 
 struct ChainNodeViewModel {
     let chainNode: ChainNode
-    
+
+    private let blockNumber: BigInt?
+    private let latency: LatencyMeasureService.Latency?
+    private let blockNumberError: Error?
+
+    private static let valueFormatter = ValueFormatter.full_US
+
+    init(chainNode: ChainNode, blockNumber: BigInt?, latency: LatencyMeasureService.Latency?, blockNumberError: Error?) {
+        self.chainNode = chainNode
+        self.blockNumber = blockNumber
+        self.latency = latency
+        self.blockNumberError = blockNumberError
+    }
+
     var title: String {
-        guard let host = chainNode.host else {
-            return ""
+        guard let host = chainNode.host else { return "" }
+        return chainNode.isGemNode ? "Gem Wallet Node" : host
+    }
+
+    var titleExtra: String? {
+        let title = Localized.Nodes.ImportNode.latestBlock
+        let emptyTitle = "\(title): -"
+        if let blockNumberError {
+            return Self.isNotImplemented(error: blockNumberError) ? nil  : emptyTitle
         }
-        if host.contains("gemnodes.com") {
-            return "Gem Wallet Node"
+
+        if let blockNumber {
+            let formattedBlockNumber = ChainNodeViewModel.valueFormatter.string(blockNumber, decimals: 0)
+            return "\(title): \(formattedBlockNumber)"
         }
-        return host
+
+        return emptyTitle
+    }
+
+    var subtitle: String? {
+        if let blockNumberError {
+            return Self.isNotImplemented(error: blockNumberError) ? nil  : "\(Localized.Errors.error) 🔴"
```

Comment:
can we move `🔴` and all emojies to static vars, similar with `SystemImage`? to simplify reading and referring